src/Container.Database.MySql/Container.Database.MySql.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <RootNamespace>TestContainers.Container.Database.MySql</RootNamespace>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <AssemblyName>TestContainers.Container.Database.MySql</AssemblyName>
+    <Description>DotNet port of testcontainers.org</Description>
+
+    <Version>1.4.9</Version>
+    <Authors>Isen Ng</Authors>
```

Comment:
Would you like to add your name to the list of authors?
You can add it as a comma separated list.